Modules/selectmodule.c

```diff
@@ -58,6 +58,13 @@ extern void bzero(void *, int);
 #  define SOCKET int
 #endif
 
+#if defined(MS_WINDOWS)
+#define FD_SET_WIN(fd, set, setsize) do { \
```

Comment:
Please use a regular function here instead of macros.
The function may be `static inline` if you really care about speed (personally I don't think that `select` with many sockets can be performant at all).
@vstinner wants to get rid of macros in CPython, let's don't add a new one.